OpenRA.Mods.Common/Scripting/Global/UserInterfaceGlobal.cs

```diff
@@ -35,7 +35,7 @@ public void SetMissionText(string text, Color? color = null)
 
 		[Desc("Formats a language string for a given string key defined in the language files (*.ftl). " +
 			"Args can be passed to be substituted into the resulting message.")]
-		public string Translate(string key, [ScriptEmmyTypeOverride("{ string: any }")] LuaTable args = null)
+		public string FluentMessage(string key, [ScriptEmmyTypeOverride("{ string: any }")] LuaTable args = null)
```

Comment:
I think the previous API name is explaining things better to scripters.

Reply:
I think GetFluentMessage is good enough

Reply:
Ok, renamed.